required/latex-lab/latex-lab-table.dtx

```diff
@@ -1015,26 +1043,26 @@
  }
 %    \end{macrocode}
 %  \end{macro}
-
+%
 %    \begin{macrocode}
 % See tagpdfsetup-keys.md in tagpdf/doc for the naming scheme.
 \keys_define:nn { __tag / setup }
   {
     table/tagging .choices:nn = { true, on }
-      { \__tag_tbl_enable: },
+      { 
+        \__tag_tbl_enable: 
+      },
     table/tagging .choices:nn = { false, off }
       { \__tag_tbl_disable: },
     table/tagging .choice:,
     table/tagging / presentation .code:n = 
-     {
-       \__tag_tbl_enable:
-       \tl_set:Nn\l_@@_rowtag_tl   {NonStruct}
-       \tl_set:Nn\l_@@_pcelltag_tl {NonStruct}
-       \tl_set:Nn\l_@@_celltag_tl  {text}
-       \tl_set:Nn\l_@@_tabletag_tl {Div} 
-       \clist_clear:N \l_@@_header_rows_clist
-       \clist_clear:N \l_@@_header_columns_clist  
-     },  
+      {
+        \__tag_tbl_enable:
+        \AssignSocketPlug{tagsupport/tbl/hmode/begin}{LayoutTable}
+        \AssignSocketPlug{tagsupport/tbl/vmode/begin}{LayoutTable}
+        \clist_clear:N \l_@@_header_rows_clist
+        \clist_clear:N \l_@@_header_columns_clist  
```

Comment:
If I go from presentation back to table/tagging=on it should probably reset the plugs or not? If so the plugs have to be set also above, as this this might happen on the same group level.

If my assumption is correct, then this also needs a test case.

Reply:
`\__tag_tbl_enable:`  sets all sockets, including the begin sockets to the defaults. So the presentation mode only has to overwrite the two.

Reply:
and there is a testfile (table-015-presentation in testfiles-table-luatex) which tests the switch from presentation back to normal tagging in one group.